src/foam/nanos/auth/resetPassword/ResetView.js

```diff
@@ -167,9 +176,9 @@ foam.CLASS({
         .start().addClass('Reset-Password').add("Reset Password").end()
         .start().addClass('Message-Container')
           .start().addClass('newPassword-Text').add("New Password").end()
-          .start(this.NEW_PASSWORD).addClass('full-width-input').end()
+          .start(this.NEW_PASSWORD).end()
           .start().addClass('confirmPassword-Text').add("Confirm Password").end()
-          .start(this.CONFIRM_PASSWORD).addClass('full-width-input').end()
+          .start(this.CONFIRM_PASSWORD).end()
```

Comment:
You can just do .add(this.CONFIRM_PASSWORD), which is shorter than start() and end().